app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(players: list[Elf]) -> None:
```

Comment:
we know that we expect a list of elves/dwarves in below functions, so you could use a better name for this and below variables. Players is too general